src/f32/quat.rs

```diff
@@ -454,8 +454,25 @@ impl Quat {
         self.mul_vec3a(Vec3A::from(other)).into()
     }
 
+    #[inline]
+    /// Adds two quaternions.
+    /// The sum not guaranteed to be normalized.
+    /// NB: Addition is not the same as combining the rotations represented by the two quaternions!
+    /// That corresponds to multiplication.
+    pub fn add_quat(self, other: Self) -> Self {
```

Comment:
I haven't been super consistent about adding functions as well as operators, but generally when I've added them it has been so there is a call by `&self` version, in particular for matrices which can be large. I think in this case it's not necessary and adding the implementation and comment on the operator itself would be sufficient.

Reply:
I added them mainly because I was following the example of mul_quat, and because I didn't think doc comments showed up in the generated docs. I was surprised to find they do, you just have to expand the trait impl to see them.

Reply:
Done!